CHANGELOG

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [v0.5] 16 February, 2022
```

Comment:
Won't we just add back an `## [Unreleased]` tag on the next addition not-yet-formally-released addition to the codebase?

Reply:
I will on another commit and PR :)